### PR TITLE
Signs are no longer affected by tesla zaps

### PR DESCRIPTION
**Pull request by Bawhoppen**
if only there was a better way to list all the things

### CODE REVIEW

File: code/modules/power/tesla/energy_ball.dm [hunk 1]
```diff
@@ -14,6 +14,38 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 										/obj/structure/particle_accelerator/end_cap,
 										/obj/machinery/field/containment,
 										/obj/structure/disposalpipe,
+										/obj/structure/sign,
```
Comment (KazeEspada) [inline code/modules/power/tesla/energy_ball.dm]: Pretty sure this list works for all children, I mean just look at the simple animal entry.

Comment (Bawhoppen) [inline code/modules/power/tesla/energy_ball.dm]: goof said it did not, and look at the PA entry

Comment (RemieRichards) [inline code/modules/power/tesla/energy_ball.dm]: Goof would be wrong, @Bawhoppen , it uses ```is_type_in_list()``` so it works on subtypes, just ```/obj/structure/sign``` will suffice.

Comment (Bawhoppen) [inline code/modules/power/tesla/energy_ball.dm]: guess this is what I deserve for listening to goof

Comment (KazeEspada) [inline code/modules/power/tesla/energy_ball.dm]: Do your own research.

**Pull request merged**